library/include/modules/Units.h

```diff
@@ -115,7 +115,7 @@ DFHACK_EXPORT df::unit_misc_trait *getMiscTrait(df::unit *unit, df::misc_trait_t
 DFHACK_EXPORT bool isDead(df::unit *unit);
 DFHACK_EXPORT bool isAlive(df::unit *unit);
 DFHACK_EXPORT bool isSane(df::unit *unit);
-DFHACK_EXPORT bool isCitizen(df::unit *unit);
+DFHACK_EXPORT bool isCitizen(df::unit *unit, bool ignore_sanity = false);
```

Comment:
how do the behavior of `isUndead` and `isCitizen` change when called from Lua? Does it respect the default parameter value? You might need to write a wrapper function that handles the optional params.

Reply:
and update the Lua API docs

Reply:
Yea, it works as expected. I've tested via
```lua
[lua]# u = dfhack.gui.getSelectedUnit()
[lua]# print(u.mood)
-1
[lua]# print(dfhack.units.isCitizen(u))
true
[lua]# u.mood = 7
[lua]# print(dfhack.units.isCitizen(u))
false
[lua]# print(dfhack.units.isCitizen(u, false))
false
[lua]# print(dfhack.units.isCitizen(u, true))
true
```

Reply:
> and update the Lua API docs

you'll have to let me know if what I've done is sufficient here